docs/source/index.rst

```diff
@@ -1,11 +1,60 @@
 ipywidgets
 ==========
 
-Full Table of Contents
-----------------------
+.. only:: html
+
+    :Release: |release|
+    :Date: |today|
+
+**ipywidgets**, also known as jupyter-widgets or simply widgets, are
+`interactive HTML widgets <https://github.com/jupyter-widgets/ipywidgets/blob/master/docs/source/examples/Index.ipynb>`_
+for Jupyter notebooks and the IPython kernel.
+
+Notebooks come alive when interactive widgets are used. Users gain control of
+their data and can visualize changes in the data.
+
+Learning becomes an immersive, fun experience. Researchers can easily see
+how changing inputs to a model impact the results. We hope you will add
+ipywidgets to your notebooks, and we're here to help you get started.
```

Comment:
Perhaps remove the usage of `we` here. Per the [jupyterlab docs guidelines](https://jupyterlab.readthedocs.io/en/stable/developer/contributing.html#writing-style)

> The documentation should be written in the second person, referring to the reader as “you” and not using the first person plural “we.” The author of the documentation is not sitting next to the user, so using “we” can lead to frustration when things don’t work as expected.

Reply:
Perhaps also say something about discourse + gitter + stackoverflow tag

Reply:
Sure I can change that. It was just copy pasted from the README.